package.json

```diff
@@ -4,43 +4,125 @@
   "description": "",
   "main": "src/index.js",
   "scripts": {
-    "build": "wp-scripts build && npm run theme",
-    "format:js": "wp-scripts format-js",
-    "lint:js": "wp-scripts lint-js",
-    "packages-update": "wp-scripts packages-update",
-    "start": "wp-scripts start",
+    "build": "wds-build && npm run theme",
```

Comment:
You'll notice new things in here, like `wds-*` commands, see https://github.com/WebDevStudios/npm-scripts/ for an explanation!